source/system/external/windows.ooc

```diff
@@ -7,7 +7,7 @@
  */
 
 version(windows) {
-include windows | (_WIN32_WINNT=0x0500)
+include windows | (NOMINMAX, _WIN32_WINNT=0x0500)
```

Comment:
I remember a funny compilation error related to some windows headers being included in wrong order, so yes, this does not surprise me...